utils/credentials.go

```diff
@@ -15,6 +15,9 @@ var (
 	ClientSecret = "clientSecret"
 	WorkspaceRef = "workspaceRef"
 	Provider     = "provider"
+	Username     = "userName"
```

Comment:
the string should be `username`

Reply:
I prefer it that way too. 
But I noticed somewhere in the code to be written that way. Focusing on `how`, not at `what`.

Reply:
![image](https://github.com/user-attachments/assets/af02f047-8dbc-4fea-89e6-dc5ee589d89d)